### PR TITLE
Business dashboard / categories / list screens

### PR DESCRIPTION
**Pull request by olbesp**
It includes:
BusinessDashboard screen,
BusinessCategories screen,
BusinessList screen,
separate ServiceCategories component,
separate Promotion component,
new ArrowList & ArrowItem components,
new ButtonBlock component,
new BusinessCard component
image 2

### CODE REVIEW

**Comment by LiorBraginsky** [comment]
в [ArrowList](https://github.com/fulcrumi/aiirush-rn/pull/28/commits/1e1b24ca3c4836cbdf52e2537a796d02d49dd121)  я бы поменял `flex: 1` на  `paddingHorizontal: 20`
 флекс его растягивает на пол страницы

File: src/components/ServiceCategories/ServiceCategories.js [hunk 1]
```diff
@@ -0,0 +1,63 @@
+import React from 'react'
```
Comment (iarickvigasi) [inline src/components/ServiceCategories/ServiceCategories.js]: You may want to rename .js file to index.js as it is the main file

File: src/components/UI/Promotion.js [hunk 2]
```diff
@@ -0,0 +1,76 @@
...
+      />
+    </View>
+    <View style={styles.introContainer}>
+      <Text style={styles.introTitle}> Introducing AiiR Service </Text>
+      <Text style={styles.introSubTitle}> A way to complete your everyday needs,{'\n'} we got this for you </Text>
+    </View>
+  </View>
+)
+
+export default Promotion
```
Comment (iarickvigasi) [inline src/components/UI/Promotion.js]: Thanks for moving :)
👍

File: src/screens/BusinessCategories/index.js [hunk 3]
```diff
@@ -0,0 +1,64 @@
...
+    return (
+      <Container style={styles.container}>
+        <ScrollView>
+          <View style={styles.header}>
+            <Title>Create New Business</Title>
+            <SubTitle>Choose category - Create your business</SubTitle>
+          </View>
+          <ServiceCategories
+            businessProfile
+            aiirBeautyPressHandler={this._onBeautyPress}
```
Comment (iarickvigasi) [inline src/screens/BusinessCategories/index.js]: It is a bad practice to include project name to the code

Comment (iarickvigasi) [reply src/screens/BusinessCategories/index.js]: Better to provide one handler and pass some identificator

File: src/screens/Services/index.js [hunk 4]
```diff
@@ -1,13 +1,18 @@
 import * as React from 'react'
 import { ScrollView, Image, TouchableOpacity } from 'react-native'
 import { Container, View, Icon, Text } from 'native-base'
+import Promotion from '/components/UI/Promotion'
+import Title from '../../components/Typography/Title'
+import SubTitle from '../../components/Typography/Title'
```
Comment (iarickvigasi) [inline src/screens/Services/index.js]: try to use absolute urls 
like this - /components/...

File: src/screens/Services/index.js [hunk 5]
```diff
@@ -18,86 +23,46 @@ class Services extends React.Component<Props, State> {
...
+
+	_onMartPress = () => {
+		this.props.navigation.navigate('Soon');
+	}
+
+	_onFoodPress = () => {
+		this.props.navigation.navigate('Restaurants');
+	}
+	
+	_onProsPress = () => {
```
Comment (iarickvigasi) [inline src/screens/Services/index.js]: We may change to one handler, 
Though it is not in plans to extend this functionality

**Comment by olbesp** [comment]
Fixed

**Pull request merged**